scratchapi2/user.py

```diff
@@ -138,96 +161,81 @@ def info(self):
         self.about_me = self.status
         self.what_im_working_on = self.bio
         self.what_working_on = self.bio
-        return self.__dict__
+        return self.__dict__.copy()

```

Comment:
`return self.__dict__` will return a special kind of `dict` that allows other functions to change the attributes of the instance, even if the variable reference is disconnected! `.copy()` ensures it's just a copy so that the original cannot be modified.